exchanges/stream/websocket.go

```diff
@@ -488,15 +509,26 @@ func (w *Websocket) trafficMonitor() {
 				w.setTrafficMonitorRunning(false)
 				w.Wg.Done()
 				return
-			case <-w.TrafficAlert:
-				if !trafficTimer.Stop() {
-					select {
-					case <-trafficTimer.C:
-					default:
+			case t := <-w.TrafficAlert:
+				if w.AuthConn != nil && w.AuthConn.GetURL() == t {
+					if !trafficAuthTimer.Stop() {
+						select {
+						case <-trafficAuthTimer.C:
+						default:
```

Comment:
Can be changed to: 
```go
<-trafficTimer.C
```

Reply:
It's a two different timer. In that case logic will work wrong, I think.

Reply:
whoops, you are correct what I meant was `<-trafficAuthTimer.C` 🤭

Reply:
Yeah, we can remove the select case. As it shouldn't be needed.